docs/tutorial_hist.rst

```diff
@@ -1,13 +1,8 @@
 .. _tutorial_hist:
 
-************************************
-Tutorial: History
-************************************
```

Comment:
I use a tool called Leo to edit source code and it auto imports different file types and cuts them up. Apparently it removed the headings...